plugin/registration/milxQtRegistrationImage.cpp

```diff
@@ -0,0 +1,305 @@
+#include "milxQtRegistrationImage.h"
```

Comment:
Can you rename this class to just milxQtRegistration? We may want to create a milxQtRegistrationImage class later that overloads the milxQtImage class to add custom registration enable menus etc.